cypress/integration/dashboard-spec.js

```diff
@@ -3,21 +3,86 @@ describe('Dashboard UI', () => {
     cy.visit('http://localhost:3000');
   });
 
-  it('Should confirm that true is equal to true', () => {
-    expect(true).to.equal(true)
+  it('Should be able to visit the page and render the title', () => {
+    cy.get('header').contains('Rancid Tomatillos').should('be.visible')
   });
 
-  it('Should be able to visit the page and render the correct elements', () => {
-      cy.contains('Rancid Tomatillos')
-      cy.contains('Top Rated')
-      cy.get('footer').contains('About')
+  it('Should render a section for the Top Rated movies that has the top three movies\'s titles and ratings', () => {
+    cy.contains('Top Rated')
+    cy.get('.top-three-section > #1').contains('7')
+    cy.get('.top-three-section > #2').contains('7')
+    cy.get('.top-three-section > #3').contains('6.8')
+    cy.get('.top-three-section > #1').contains('h4', 'Peninsula')
+    cy.get('.top-three-section > #2').contains('h4', 'Cats & Dogs 3: Paws Unite')
+    cy.get('.top-three-section > #3').contains('h4', 'The King of Staten Island')
```

Comment:
This looks really great! Has me wondering though how to do this with a set of movies/ratings that is dynamic.